celery/backends/new_cassandra.py

```diff
@@ -0,0 +1,199 @@
+# -* coding: utf-8 -*-
+"""
+    celery.backends.new_cassandra
+    ~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Apache Cassandra result store backend using DataStax driver
+
+"""
+from __future__ import absolute_import
+
+import sys
+import six
+try:  # pragma: no cover
+    import cassandra
+except ImportError:  # pragma: no cover
+    cassandra = None   # noqa
+
+from celery import states
+from celery.exceptions import ImproperlyConfigured
+from celery.utils.log import get_logger
+from .base import BaseBackend
+
+__all__ = ['NewCassandraBackend']
+
+logger = get_logger(__name__)
+
+
+class NewCassandraBackend(BaseBackend):
+    """New Cassandra backend utilizing DataStax driver
+
+    .. attribute:: servers
+
+        List of Cassandra servers with format: ``hostname``
+
+    :raises celery.exceptions.ImproperlyConfigured: if
+        module :mod:`cassandra` is not available.
+
+    """
+    servers = []
```

Comment:
Please, avoid mutable class-level objects. IMO we don't need all these connection-related parameters on class level.